multi_import/fields.py

```diff
@@ -0,0 +1,50 @@
+from rest_framework import relations
+
+from multi_import.utils import normalize_string
+
+__all__ = [
+    'FieldHelper',
+]
+
+
+class FieldHelper(object):
+    list_separator = ';'
+
+    def to_string_representation(self, field, value):
+        if hasattr(field, 'to_string_representation'):
+            return field.to_string_representation(value)
+
+        if isinstance(field, relations.ManyRelatedField):
+            return self.many_to_string_representation(field, value)
+
+        if value is None:
```

Comment:
I don't want to override all falsey values though. False and 0 are valid values.

I think this might be the case for everywhere I explicitly check for None.